src/components/region-common.js

```diff
@@ -1,74 +1,77 @@
 // Get a { flag: ..., countryName: ... } object representing a google locationType/location input.
 // 'flag' will always be defined (even if it's a question mark.
 // 'regionDescription' is the same as location when locationType is 'multi-region', or a country name when locationType is 'region'.
+// computeZone is generally the 'a' zone for each region, except for those regions where it is not available.
+// The choice to use the 'a' zone is arbitrary, choosing 'b' zone would also work.
+// The region choice for multi-region locations is arbitrary as well.
 export const unknownRegionFlag = '❓'
 export const regionInfo = (location, locationType) => {
   switch (locationType) {
     case 'multi-region':
       switch (location) {
```

Comment:
we have a utility function `Utils.switchcase` (in `utils.js`) to clean up the syntax of switches a bit